manifests/00-custom-resource-definition.yaml

```diff
@@ -705,9 +361,6 @@ spec:
                     type: string
                   type:
                     type: string
-                required:
-                - type
-                - status
```

Comment:
this looks suspicious.

Reply:
cc @Miciah

Reply:
@sttts Isn't this from https://github.com/openshift/api/blob/master/operator/v1/types.go#L148? They aren't `omit empty` and aren't explicitly marked required (the package default is optional). Is something wrong with the shared OperatorCondition struct's markers?

Reply:
Discussion in https://github.com/openshift/api/pull/445